int32 nbSeg
int32 unsused
SM_SEGMENT_STR_ROS[] seg

